tests/testthat/test-bind.R

```diff
@@ -862,8 +862,8 @@ test_that("c() fallback works with unspecified columns", {
 
```

Comment:
Line 835 has this skip, can we remove it? `skip("FIXME: c() fallback with recursion through df-col")`

Reply:
Good catch.